app/tasks/raster_tile_cache_assets.py

```diff
@@ -0,0 +1,423 @@
+        "-v",
+        version,
+        "-j",
+        json.dumps(jsonable_encoder(co_copy)),
+    ]
+
+    if overwrite:
+        command += ["--overwrite"]
+
+    if subset:
+        command += ["--subset", subset]
+
+    pixetl_job_id = PixETLJob(
+        job_name=job_name,
+        command=command,
+        environment=JOB_ENV,
+        callback=callback,
+        parents=[parent.job_name for parent in parents] if parents else None,
+    )
+
+    return pixetl_job_id
+
+
+async def _reproject_to_web_mercator(
+    dataset: str,
+    version: str,
+    source_creation_options: Dict[str, Any],
+    min_zoom: int,
+    max_zoom: int,
+    parents: Optional[List[Job]],
+) -> List[Job]:
+    reprojection_jobs: List[Job] = []
+
+    # Processing chokes on large datasets if we go directly to low
+    # zoom levels, so start at the highest and work our way back
+
+    co = copy.deepcopy(source_creation_options)
+    co["resampling"] = "med"
+    co["overwrite"] = co.get("overwrite", False)
+    co["srid"] = "epsg-3857"
+
+    for zoom_level in reversed(range(min_zoom, max_zoom)):
+        co["grid"] = f"zoom_{zoom_level}"
+
+        asset_uri = get_asset_uri(dataset, version, AssetType.raster_tile_set, co)
+        del co["srid"]
+
+        co_obj = RasterTileSetSourceCreationOptions(**co)
+
+        # Create an asset record
+        asset_options = AssetCreateIn(
+            asset_type=AssetType.raster_tile_set,
+            asset_uri=asset_uri,
+            is_managed=True,
+            creation_options=co_obj,
+            metadata={},
+        ).dict(by_alias=True)
+        wm_asset_record = await create_asset(dataset, version, **asset_options)
+        print(f"ZOOM LEVEL {zoom_level} REPROJECTION ASSET CREATED")
+
+        zoom_level_job = await _run_pixetl(
+            dataset,
+            version,
+            wm_asset_record.creation_options,
+            f"zoom_level_{zoom_level}_{co['pixel_meaning']}_reprojection",
+            callback_constructor(wm_asset_record.asset_id),
+            parents=(parents + reprojection_jobs) if parents else reprojection_jobs,
+        )
+        reprojection_jobs.append(zoom_level_job)
+        print(f"ZOOM LEVEL {zoom_level} REPROJECTION JOB CREATED")
+
+        co["srid"] = "epsg-3857"
+        source_uri = get_asset_uri(
+            dataset, version, AssetType.raster_tile_set, co
+        ).replace("{tile_id}.tif", "tiles.geojson")
+        co["source_uri"] = [source_uri]
+
+    return reprojection_jobs
+
+
+async def _merge_intensity_and_date_conf(
+    dataset: str,
+    version: str,
+    date_conf_co: Dict[str, Any],
+    intensity_co: Dict[str, Any],
+    min_zoom: int,
+    max_zoom: int,
+    parents: List[Job],
+):
+    merge_intensity_jobs: List[Job] = []
+
+    for zoom_level in range(min_zoom, max_zoom):
+        # Sanitize creation_options
+
+        d_c_co = copy.deepcopy(date_conf_co)
+        d_c_co["srid"] = "epsg-3857"
+        d_c_co["grid"] = f"zoom_{zoom_level}"
```

Comment:
You don't use `date_conf_co` and `intensity_co` elsewhere. You don't need to use deepcopy, do you?
since srid is not part of the model, i would keep it seperate. In make it a separate argument in `get_asset_uri`  and have it default to `epsg:4326`.
Please note that the actual srid notation has a colon `:` not a dash `-`. I replaced it with the dash b/c otherwise it would conflict with standard path naming conventions. In `get_asset_uri` you will want to sanitize the srid argument and replace colon with dash